packages/truffle/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "app-module-path": "^2.2.0",
-    "mocha": "5.2.0",
+    "mocha": "https://github.com/trufflesuite/mocha.git#v5.2.1",
```

Comment:
This will force users to have git to install Truffle.

Also, don't do this :)

Reply:
Good point. `@truffle/mocha` coming right up. 👏

Reply:
No but please don't do it this way 🙌

Reply:
@CruzMolina did can this just say "5.2.1" instead of "5.2.0" or was there a reason you did it this way?

Reply:
@fainashalts Currently `5.2.1` only exists as a branch on our fork. Want me to publish it as `@trufflesuite/mocha`?